### PR TITLE
docs: Name change

### PR DESCRIPTION
**Pull request by connerleverett**
Changing Geocortex Workflow to VertiGIS Studio Workflow and apps.geocortex to apps.vertigisstudio

### CODE REVIEW

**Comment by connerleverett** [comment]
To be merged after Feb 14 2022

File: README.md [hunk 1]
```diff
@@ -1,13 +1,13 @@
 [![CI/CD](https://github.com/geocortex/workflow-activities-maximo/workflows/CI/CD/badge.svg)](https://github.com/geocortex/workflow-activities-maximo/actions)
 [![npm](https://img.shields.io/npm/v/@geocortex/workflow-activities-maximo)](https://www.npmjs.com/package/@geocortex/workflow-activities-maximo)
 
-This project contains activities for interacting with the **Maximo REST API** in a [Geocortex Workflow](https://www.geocortex.com/products/geocortex-workflow/).
+This project contains activities for interacting with the **Maximo REST API** in a [VertiGIS Studio Workflow](https://www.vertigisstudio.com/products/geocortex-workflow/).
```
Comment (kenlyon) [inline README.md]: Should the `/geocortex-workflow/` ending of the url also change?

File: package.json [hunk 2]
```diff
@@ -16,7 +16,7 @@
         "Maximo",
         "IBM",
         "ArcGIS",
-        "GIS"
+        "GIS"
```
Comment (kenlyon) [inline package.json]: Extra spaces added for some reason.

**Review by kenlyon — CHANGES_REQUESTED** [review]

File: package.json (hunk 2 above)
Comment (connerleverett) [reply package.json]: Removed (and updated Geocortex to VertiGIS Studio)

File: README.md (hunk 1 above)
Comment (connerleverett) [reply README.md]: Updated

**Review by kenlyon — APPROVED** [review]

**Pull request merged**